packages/concerto-tools/lib/codegen/fromJsonSchema/cto/jsonSchemaVisitor.js

```diff
@@ -772,6 +802,19 @@ class JsonSchemaVisitor {
             return this.visitJsonSchemaModel(thing, parameters);
         }
     }
+    /**
+     * Create a JSON Schema model class, used to start the inference into
+     * Concerto JSON.
+     * @param {Object} jsonSchemaModel - the JSON Schema Model.
+     *
+     * @return {Object} the result of visiting or undefined.
+     * @public
+     */
+    static infer(jsonSchemaModel) {
```

Comment:
`infer` doesn't sound right to me, it's the visitor that's doing the inference. This is a little like a factory method. You could call it `deserialize` or `parse` for example?

Reply:
Addressed ✅